### ISSUE TITLE
"Enable client and server targets by default in 4.23.0 onwards" only applied to Windows

### ISSUE THREAD

**Issue by TBBle** [adamrehn/ue4-docker#70]
I was quite excited to take advantage of 91f4381aa9c030ad6431f4c7f4dc0510219dbdd6 and UE 4.23, but I only just noticed the patch was only applied to the Windows image, when my Linux Server builds failed.

;_;

I'm going to locally try and hack this into the Linux build locally, and if it works out, will put up a PR. Of course, I won't mind if someone else gets there first. ^_^

**Comment by adamrehn** [comment]
Oops, how did I miss this? Fixed and live in ue4-docker version 0.0.53.

**Comment by TBBle** [comment]
Fix looks good using `docker cp` against the container running the build step (rather than waiting overnight for the actual images...)
```
paulh@docker-test:~$ docker cp 1d8e8ac2ed61:/home/ue4/UnrealEngine/Engine/Build/InstalledEngineBuild.xml - |grep -a 'Set up target types' -A 2
        <!-- Set up target types -->
        <Option Name="WithClient" Restrict="true|false" DefaultValue="true" Description="Include precompiled client targets"/>
        <Option Name="WithServer" Restrict="true|false" DefaultValue="true" Description="Include precompiled server targets"/>
```
